Tests/ApodiniTests/SemanticModel/EnrichedInfoTests.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  Created by Lorena Schlesinger on 1/15/21.
+//
+
+import XCTest
+@_implementationOnly import Runtime
+
+@testable import Apodini
+
+class EnrichedInfoTests: ApodiniTests {
+    func testCardinalityIsEquatable() throws {
```

Comment:
This is not hard evidence, but I feel these tests are very fine-grained

Reply:
That's right, I got a bit incited: https://github.com/Apodini/Apodini/pull/143#issuecomment-761889533 😄